src/pykeen/losses.py

```diff
@@ -953,6 +954,68 @@ def forward(
         return loss
 
 
+@parse_docdata
+class FocalLoss(PointwiseLoss):
+    """
+    Implementation of the focal loss as a module.
+
+    Inspired by the functional form from torchvision.
+
+    Proposed by [lin2018]_.
+
+    # TODO: Move to correct place
+    Focal Loss for Dense Object Detection
+    Tsung-Yi Lin, Priya Goyal, Ross Girshick, Kaiming He, Piotr Dollár
+    ICCV'17.
+    https://arxiv.org/abs/1708.02002
```

Comment:
@cthoyt where do we collect the references for the loss modules?

Reply:
See 269400aa

Reply:
thanks! :+1: